sd-card/config/config.ini

```diff
@@ -62,6 +62,9 @@ CheckDigitIncreaseConsistency = false
 RetainMessages = false
 HomeassistantDiscovery = false
 ;MeterType = other
+;CACert = /config/certs/filepath
```

Comment:
I would add default file names instead of just `filepath`.

Reply:
updated with more specfici names (yet still a bit generic)